auth-web/src/components/auth/account-settings/statement/Statements.vue

```diff
@@ -178,6 +186,15 @@ export default defineComponent({
       }
     ])
 
+    const isStatementNew = (item) => {
+      return Math.max(...statementsList.value.map(statement => statement.id)) === item.id
+    }
+
+    const isStatementOverdue = (item) => {
+      return (paymentOwingAmount.value && paymentDueDate.value) &&
```

Comment:
Would this show them all as overdue? - do we need to change the summary endpoint to show which are overdue?

Reply:
sounds good I'll fix this on the BE